charts/opensearch/ci/ci-values.yaml

```diff
@@ -320,12 +322,21 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+startupProbe:
+  enabled: false
+  tcpSocket:
+    port: 9200
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 3
+  failureThreshold: 30
+
 readinessProbe:
   failureThreshold: 3
-  initialDelaySeconds: 10
+  initialDelaySeconds: 900
```

Comment:
Is 15 minutes too long to fire in CI? Maybe there should also be a test using the default values?

Reply:
I have seen it take this much time for a 50 node cluster which I was trying to bring up. We can bring down the value, it should not be an issue

Reply:
@smlx Are we good to close this comment?

Reply:
Yep!